tests/unit/accounts/test_views.py

```diff
@@ -222,11 +224,10 @@ def test_post_validate_no_redirects(self, pyramid_request,
             username=pretend.stub(data="theuser"),
         )
         form_class = pretend.call_recorder(lambda d, user_service: form_obj)
-
+        pyramid_request.route_path = pretend.call_recorder(lambda a: observed_next_url)
```

Comment:
Let's make this:
```
pyramid_request.route_path = pretend.call_recorder(lambda a: '/the-redirect')
```
Just to ensure that this is only being called in the second test case.

Reply:
Okay!

Reply:
addressed :+1: